libs/application-generic/src/schemas/control/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './in-app-control.schema';
```

Comment:
These schemas are strictly connected to the built-in Dashboard UI powered by the new Novu workflow V2 API. Do they belong in the application generic? I don't feel strongly about it, but I'd consider it a code smell if these schemas were reused in other services.

Reply:
Interesting and valid point, i can share why they are in app-gen. 
The are in app-gen because i reused the types from the schema in the dashboardSanitizeControlValues function so i needed them close to it, and dashboardSanitizeControlValues is used in Worker.
Let me know what you think about such an implementation.